src/mysql_vm_helpers.py

```diff
@@ -755,7 +771,7 @@ def snap_service_operation(snapname: str, service: str, operation: str, enable=F
             selected_snap.start(services=[service], enable=enable)
             return selected_snap.services[service]["active"]
         else:
-            selected_snap.stop(services=[service])
+            selected_snap.stop(services=[service], disable=disable)
```

Comment:
feel free to ignore it: why not?
```suggestion
            selected_snap.stop(services=[service], disable=not enable)
```

Reply:
If we do `not enable`, we would always need to specify enable if we would like to avoid disabling the service when stopping it, which seems a bit counter intuitive

e.g.
```
snap_service_operation(snap_name, service_name, "stop", enable=True)
``

Reply:
removed the `enable` and `disable` kwargs in 9f7d5c199f5e6824c0f44babf3c7917c58930592 (and always enable upon start and always disable upon stop)